helix-term/src/commands/typed.rs

```diff
@@ -2529,6 +2424,17 @@ fn read(cx: &mut compositor::Context, args: &[Cow<str>], event: PromptEvent) ->
     Ok(())
 }
 
+fn echo(cx: &mut compositor::Context, args: Args, event: PromptEvent) -> anyhow::Result<()> {
+    if event != PromptEvent::Validate {
+        return Ok(());
+    }
+
+    let args = args.collect::<Vec<&str>>().join(" ");
```

Comment:
```suggestion
    let args = args.raw();
```

This returns whatever was given to `Args::from`, so no need to the allocations.

Reply:
fixed it. but i still need to clone it as editor's lifetime is bigger than args's one.